test/instrumentation/testdata/stable-metrics-list.yaml

```diff
@@ -88,6 +88,35 @@
   help: The count of hidden metrics.
   type: Counter
   stabilityLevel: BETA
+- name: certificate_manager_server_rotation_seconds
+  subsystem: kubelet
+  help: Histogram of the number of seconds the previous certificate lived before being
+    rotated.
+  type: Histogram
```

Comment:
Is histogram the right way to measure this? The choice seems a little odd to me.

Reply:
I can't really answer yes or no to this. I mainly bumped this metrics which have been in alpha for a long while to beta.